content/docs/apps/continuous-deployment.md

```diff
@@ -8,7 +8,12 @@ title: Continuous deployment
 Setting up continuous deployment allows you to automatically upload your
 changes to your desired environment.
 
-Before setting this up, go through the [Production Ready Guide]({{< relref "production-ready.md" >}}) and make sure you're following those core best practices.
+Before setting this up, go through
+
+1. Setting up continuous integration
+    * This will protect you from deploying a broken application.
+    * You can use the same service for continuous integration+deployment — see list [below]({{< relref "#continuous-integration-services" >}}) for suggestions.
+1. The [Production Ready Guide]({{< relref "production-ready.md" >}})
```

Comment:
I would reverse the order of those.. Production Ready gets you ready for CI, by pointing out using a manifest, etc.